file_optical.c

```diff
@@ -250,7 +251,7 @@ static int fbo_emulate_request_sense(struct tcmu_device *dev, uint8_t *cdb,
 		buf[12] = 0x04;		// Not Ready
 		buf[13] = 0x04;		// Format in progress
 		buf[15] = 0x80;
-		*(uint16_t *)&buf[16] = htobe16(state->format_progress);
+		put_unaligned_be16(htobe16(state->format_progress), &buf[16]);
```

Comment:
Sorry. I just noticed this, but is this right? It seems we do a conversion twice in the patch? Can't we just do

put_unaligned_be16(state->format_progress, &buf[16]);

? The put call swaps it to BE, so there is not need for the htobe16 call is there?

I will this up in the patch when I merge.

I also added get calls, so no need for the memcpy and be16toh calls.